### ISSUE TITLE
Node-inspector crashed process: TypeError: Cannot read property '1' of null

### ISSUE THREAD

**Issue by AgentME** [node-inspector/node-inspector#518]
I found a long-running process that was run with node-inspector had crashed with this stacktrace:

```
TypeError: Cannot read property '1' of null
    at getCallerFuncLocation (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:107:15), <anonymous>:63:31)
    at new ConsoleMessage (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:107:15), <anonymous>:10:20)
    at Console.debug.register.messageId [as log] (eval at WRAPPED_BY_NODE_INSPECTOR (/usr/lib/node_modules/node-inspector/node_modules/v8-debug/v8-debug.js:107:15), <anonymous>:103:21)
... < stack trace in my application code on a line that was just console.log('some string'); >
```

node-inspector 0.8.1

**Comment by 3y3** [comment]
Will be fixed in 0.8.2 releas

**Comment by 3y3** [comment]
0.8.2 released
